[DCK] Remove outdated samples

yajo opened the pull request:
- `l10n-spain` was never a good sample, since only Spanish projects would have it. Since https://github.com/Tecnativa/doodba/pull/86 it's simply not needed in most cases. Devs can check the docs to know the structure. Thus, it can be removed from here.
- `checksumdir` is not used anymore in `module_auto_update` since https://github.com/OCA/server-tools/pull/1190.
- `xlsxwriter` is included in Odoo 10+.
- `requests` is included in Odoo 8+.

pedrobaeza commented:
I still want an OCA addons sample with the origin name and so on. I know that documentation is there, but this is for a copy/paste helper and good practices for names.

yajo commented:
The scaffolding is IMHO not a good place to put code snippets, mostly those that are going to be removed very likely.

With the goal to keep it as small and useful as possible, if you want I can include a link to the relevant part of the docs where you can copy a snippet from.

OTOH, I would be glad to add a snippet for the repos.yaml file in the `.vscode` folder, so you just type "repo" and all gets autofilled. That would clutter less and help more.

pedrobaeza commented:
OK for both suggestions

yajo committed:
[DCK] Remove outdated samples

- `l10n-spain` was never a good sample, since only Spanish projects would have it. Since https://github.com/Tecnativa/doodba/pull/86 it's simply not needed in most cases. Devs can check the docs to know the structure, and a vscode snippet has been added. Thus, it can be removed from here.
- `checksumdir` is not used anymore in `module_auto_update` since https://github.com/OCA/server-tools/pull/1190.
- `xlsxwriter` is included in Odoo 10+.
- `requests` is included in Odoo 8+.

yajo commented:
Now it's cool:
![peek 26-02-2019 10-41](https://user-images.githubusercontent.com/973709/53403209-7ec0bb00-39b3-11e9-9f3e-e85aa62577df.gif)

pedrobaeza commented:
Great!
